security_privacy_ln.asciidoc

```diff
@@ -0,0 +1,614 @@
+
+In summary, while some aspects of the Lightning Network's architecture suggest that it is a step forward in terms of privacy compared to Bitcoin, other properties of the protocol may make attacks on privacy easier.
+Thorough research is needed to evaluate what privacy guarantees the LN provides and improve the state of affairs.
+We are happy to report that multiple researchers teams are working on Lightning privacy.
+In part, this chapter summarizes the research papers available at the time of writing in late 2020.
+Now let us review the attacks on the Lightning Network privacy that have been described in academic literature.
+
+
+= Attacks on Lightning =
+
+Recent research describes various ways in which the security and privacy of the LN may be compromised.
+
+== Observing payment amounts
+
+One of the goals for a privacy-preserving payment system is to hide the payment amount from uninvolved parties.
+The Lightning Network is an improvement over layer-one in this regard.
+While Bitcoin transactions are broadcast in cleartext and can be observed by anyone, Lightning payments only travel through a few nodes along the payment path.
+However, intermediary nodes do see the payment amount.
+This is necessary to create a new HTLC at every hop.
+The availability of payment amounts to intermediary nodes does not present an immediate threat.
+However, an _honest-but-curious_ intermediary node may use it as a part of a larger attack.
+
+
+== Linking senders and receivers
+
+An attacker might be interested in learning the sender and/or the receiver of a payment to reveal certain economic relationships.
+This breach of privacy could harm censorship resistance, as an intermediary node could censor payments to or from certain receivers or senders.
+Ideally, linking senders to receivers should not be possible to peers other than the sender and the receiver.
+In the following, we will consider two types of adversaries: the off-path and the on-path adversary.
+An off-path adversary tries to assess the sender and the receiver of a payment without participating in the payment routing process.
+An on-path adversary can leverage any information it might gain by routing the payment of interest.
+
+First, let us consider the *off-path adversary*.
+In the first step of this attack scenario, a potent off-path adversary deduces the individual balances in each payment channel via probing (described in a subsequent section) and forms a network snapshot at time _t_.
+It then runs the attack sometime later at time _t'_ and uses the differences between the two snapshots to infer information about payments that took place by looking at paths that have changed.
+In the simplest case, if only a single payment occurred between time _t'_ and _t_, the adversary observes a single path where the balances have changed by the same amounts.
+Thus, the adversary learns everything about this payment: the sender, the recipient, and the amount.
```

Comment:
Though it's a simple model, I think this should also mention the existence of unadvertised channels as well, and the potential that a "shadow" network exists beyond the graph view of this attacker.